spec/ruby/core/exception/no_method_error_spec.rb

```diff
@@ -56,4 +56,22 @@
       e.message.match(/private method/).should_not == nil
     end
   end
+
+  it "calls receiver.inspect only when calling Exception#message" do
+    INSPECT_ERROR = Class.new(Exception)
```

Comment:
This would define a constant on `Object`.
You can use `NoMethodErrorSpecs::INSPECT_ERROR` and then define it directly in the `fixtures/common` file.

Reply:
Or simpler: use a local var and `define_method`